src/markovevaluator.cpp

```diff
@@ -61,6 +62,10 @@ NetworkState MarkovEvaluator::GetNextNetworkState() {
 	NetworkState newState = mState.DeepCopy();
 	newState = newState - reaction.reactants;
 	newState = newState + reaction.products;
+
+	std::poisson_distribution<int> timeDistribution(accFiringRate);
+	currentTime += static_cast<double>(timeDistribution(generator));
+
```

Comment:
Så nu bruger vi poisson distribution korrekt? i stedet for vores magic number

Reply:
Yup. 

The time that it takes before a reaction occurs does not depend on the specific reaction which occurs, and the same thing is now reflected in our implementation.

Reply:
Epic nice, gj